### PR TITLE
Notification refactor

### PR DESCRIPTION
**Pull request by jacobdavis1**
# housingxyz :: pull-request

## change proposal

- [x] Reformatted everything to include UpdateAction
- [x] Thoroughly tested repository, controllers to ensure basic functionality
- [x] Destroyed bugs.
- [x] Removed status as a table.

## reviewer

- @escalonn

## contributing

this project is maintained under the terms of the [Contribution Guidelines][contribution-guidelines-url].

[contribution-guidelines-url]: https://github.com/housingxyz/housingxyz/blob/master/.github/CONTRIBUTING.md 'CONTRIBUTION GUIDELINES'

### CODE REVIEW

**Comment by escalonn** [comment]
merge master and get another team lead approval

File: account/src/Revature.Account.Api/Controllers/CoordinatorAccountController.cs [hunk 1]
```diff
@@ -15,9 +15,9 @@ namespace Revature.Account.Api.Controllers
   public class CoordinatorAccountController : ControllerBase
```
Comment (gpyofavv) [inline account/src/Revature.Account.Api/Controllers/CoordinatorAccountController.cs]: Documentation could be better.

File: account/src/Revature.Account.Api/Controllers/NotificationController.cs [hunk 2]
```diff
@@ -16,9 +16,9 @@ namespace Revature.Account.Api.Controllers
   public class NotificationController : ControllerBase
```
Comment (gpyofavv) [inline account/src/Revature.Account.Api/Controllers/NotificationController.cs]: Documentation could be better.

File: account/src/Revature.Account.Api/Controllers/ProviderAccountController.cs [hunk 3]
```diff
@@ -16,9 +16,9 @@ namespace Revature.Account.Api.Controllers
   public class ProviderAccountController : ControllerBase
```
Comment (gpyofavv) [inline account/src/Revature.Account.Api/Controllers/ProviderAccountController.cs]: Documentation could be better.

File: account/src/Revature.Account.Api/Startup.cs [hunk 4]
```diff
@@ -4,7 +4,11 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 using Microsoft.OpenApi.Models;
+using Microsoft.EntityFrameworkCore;
+using Revature.Account.DataAccess;
 using Serilog;
+using Revature.Account.DataAccess.Repositories;
```
Comment (gpyofavv) [inline account/src/Revature.Account.Api/Startup.cs]: Dual import here. I know it's for convenience, but still...,

File: account/src/Revature.Account.DataAccess/Mapper.cs [hunk 5]
```diff
@@ -17,9 +17,10 @@ public Lib.Model.ProviderAccount MapProvider(Entities.ProviderAccount provider)
       return new Lib.Model.ProviderAccount
```
Comment (gpyofavv) [inline account/src/Revature.Account.DataAccess/Mapper.cs]: Documentation could be better.

File: account/src/Revature.Account.Api/Startup.cs [hunk 6]
```diff
@@ -22,6 +26,12 @@ public Startup(IConfiguration configuration)
 
     public void ConfigureServices(IServiceCollection services)
     {
+      services.AddControllers();
+      services.AddDbContext<AccountDbContext>(options =>
+                options.UseNpgsql(Configuration.GetConnectionString("AccountDb")));
```
Comment (escalonn) [inline account/src/Revature.Account.Api/Startup.cs]: indentation >:c

File: account/src/Revature.Account.Api/Startup.cs [hunk 7]
```diff
@@ -71,6 +79,14 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
       {
         endpoints.MapControllers();
       });
+
+      // Found at https://stackoverflow.com/questions/36958318/where-should-i-put-database-ensurecreated
```
Comment (escalonn) [inline account/src/Revature.Account.Api/Startup.cs]: you're going to trust 3 year old stackoverflow over nick's bleeding edge polished code from providerxyz?

**Pull request merged**